### PR TITLE
Seed project tutorial

### PR DESCRIPTION
**Pull request by sebacancinos**
- Basic set up for seed project

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/ios-objc/00-introduction.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Introduction
+description: Brief introduction to the iOS-Swift tutorials.
```
Comment (volbap) [inline articles/native-platforms/ios-objc/00-introduction.md]: Replace Swift with Objective-C 😜

**Comment by volbap** [comment]
+1 @mpaktiti

File: articles/native-platforms/ios-objc/00-introduction.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'auth0-samples/auth0-ios-objc-sample',
+  pkgBranch: 'master',
+  pkgPath: '00-introduction',
```
Comment (mpaktiti) [inline articles/native-platforms/ios-objc/00-introduction.md]: @sebacancinos The introduction seed is always named `00-Starter-Seed` -> https://github.com/auth0-samples/auth0-ios-objc-sample/tree/master/00-Starter-Seed

File: articles/native-platforms/ios-objc/00-introduction.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+}) %>
+
+### Before Starting
+
+<div class="setup-callback">
+<p>Go to the <a href="${uiAppSettingsURL}">Application Settings</a> section in the Auth0 dashboard and make sure that <b>Allowed Callback URLs</b> contains the following value:</p>
+
+<pre><code>a0${account.clientId}://\*.auth0.com/authorize</pre></code>
+</div>
+
```
Comment (mpaktiti) [inline articles/native-platforms/ios-objc/00-introduction.md:17]: @sebacancinos We need more information in this doc. As an example have a look at our [Angular2 intro doc](https://auth0.com/docs/quickstart/spa/angular2). Most should be copy/paste with minor alterations. To be more specific we need:
- a Seed & Samples paragraph (same, just update the links)
- Create an Application (same)
- Configure Callback URLs paragraph (you already have this info, just structure it this way)
- dependencies, if applicable

File: articles/native-platforms/ios-objc/00-introduction.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+#### 1. Create an application
+
+<%= include('../../_includes/_new_app') %>_
+
+![App Dashboard](/media/articles/angularjs/app_dashboard.png)
+
+#### 2. Configure your callback URLs
+
+<%= include('../_includes/_callback-url-introduction') %>
```
Comment (mpaktiti) [inline articles/native-platforms/ios-objc/00-introduction.md:82]: @sebacancinos @pablolvillar This does not exist `../_includes/_callback-url-introduction` (neither the file not the directory) and the page cannot load due to that.

**Comment by mpaktiti** [comment]
Content moved here: https://github.com/auth0/docs/pull/2328

**Pull request closed without merging**